packages/playground/package.json

```diff
@@ -12,7 +12,8 @@
   "dependencies": {
     "react": "16.11.0",
     "react-native": "0.62.2",
-    "react-native-windows": "0.0.0-master.70"
+    "react-native-windows": "0.0.0-master.70",
+    "@react-native-community/picker": "^1.5.1"
```

Comment:
I'm not a fan of tests in the repo taking a dependency on a community module, as it's bound to cause maintenance headaches, and seems unnecessary.

It looks like the only purpose of having a Picker in the tests is so we can use it to choose various options (rather than testing Picker itself).  We could use an alternate mechanism for achieving this.  I think FlatList could work here, or some other stock control that has a list and a selected item.  Is there something already in place for the test pages that come with react-native?

Reply:
Yeah thought about it a little bit more, this will probably cause issues when we have breaking changes on react-native-windows side next time. I'll see if I can find some alternatives.